src/routes/assistants/+page.svelte

```diff
@@ -24,6 +26,13 @@
 		}
 		goto(newUrl);
 	};
+
+	const onPaginationChange = (newPageIndex: number) => {
+		pageIndex = newPageIndex;
+		const newUrl = new URL($page.url);
+		newUrl.searchParams.set("p", newPageIndex.toString());
+		goto(newUrl);
```

Comment:
you can remove that too

Reply:
handled in https://github.com/huggingface/chat-ui/pull/772/commits/53ed09bc5c6a17d95dcd844deeba44fd4f7482c0